data/actionsmap/yunohost.yml

```diff
@@ -578,6 +579,22 @@ app:
                     full: --app
                     help: Write domain & path to app settings for further checks
 
+        ### app_registerurl()
+        registerurl:
```

Comment:
We discussed this IRL, but I don't think there's a mechanism at the moment in the Moulinette to 'hide' commands.